crates/turborepo-lib/src/commands/login.rs

```diff
@@ -111,16 +121,25 @@ fn make_token_name() -> Result<String> {
 
 pub async fn login(base: &mut CommandBase) -> Result<()> {
     let repo_config = base.repo_config()?;
-    let login_url_base = repo_config.login_url();
-    debug!("turbo v{}", get_version());
-    debug!("api url: {}", repo_config.api_url());
-    debug!("login url: {login_url_base}");
```

Comment:
Wrong place for this. If we want config gathering logs we should be doing it when we set the cell so that it is consistent and has _far_ more information.

We don't even directly use `api_url` in this method—transitive use only.